blueocean-github-pipeline/src/main/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubScmFileContent.java

```diff
@@ -0,0 +1,139 @@
+package io.jenkins.blueocean.blueocean_github_pipeline;
+
+import org.kohsuke.stapler.export.Exported;
+import org.kohsuke.stapler.export.ExportedBean;
+
+/**
+ * @author Vivek Pandey
+ */
+@ExportedBean
+public class GithubScmFileContent{
```

Comment:
Save operation is scm dependent. Maybe I can wrap the scm response in other element such as:

```
{
    "content":{
       "sha":"xxxx",
       ....
    }
}
```